src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService users = new UserService();
```

Comment:
```suggestion
        UserService userService = new UserService();
```
you have User model and UserService service, naming shouldn't be confusing